scilpy/utils/streamlines.py

```diff
@@ -664,3 +665,41 @@ def cut_invalid_streamlines(sft):
                                           data_per_point=new_data_per_point)
 
     return new_sft, cutting_counter
+
+
+def upsample_tractogram(
+    sft, nb, point_wise_std=None,
+    streamline_wise_std=None, gaussian=None, spline=None, seed=None
+):
+    """ Generate new streamlines by either adding gaussian noise around
+    streamlines' points, or by translating copies of existing streamlines
+    by a random amount.
+    """
```

Comment:
clean docstring

Reply:
Clean quoi ?

Reply:
I think it means to add a description for each parameters.

Reply:
Donesoes in 93351ff